module_report/__openerp__.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# © <2016> <Siyuan Gu>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{'name': 'Module Report',
+ 'version': '9.0.1.0.0',
```

Comment:
Add a summary key with:
Technical module to allow report generation based on Docx template and Jinja2